### PR TITLE
Update CookieBannerPartial.cshtml

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.Reservations.Web/Views/Shared/CookieBannerPartial.cshtml [hunk 1]
```diff
@@ -21,7 +20,7 @@
             <p class="das-cookie-banner__confirmation-message">
                 You've accepted all cookies. You can <a class="govuk-link" asp-external-controller="CookieConsent" asp-external-action="Settings" asp-external-subdomain="accounts" id="link-cookie-settings">change your cookie settings</a> at any time.
             </p>
-            <button class="das-cookie-banner__hide-button" data-hide-cookie-banner="true" id="btn-hide-cookie-banner">Hide</button>
+            <button class="das-cookie-banner__hide-button" data-hide-cookie-banner="true">Hide</button>
```
Comment (shomavg) [inline src/SFA.DAS.Reservations.Web/Views/Shared/CookieBannerPartial.cshtml:23]: Why is the id being removed?

Comment (Howesy77) [reply src/SFA.DAS.Reservations.Web/Views/Shared/CookieBannerPartial.cshtml]: I've added it back in

File: src/SFA.DAS.Reservations.Web/Views/Shared/CookieBannerPartial.cshtml [hunk 2]
```diff
@@ -4,14 +4,13 @@
...
                     <div class="das-cookie-banner__buttons">
                         <div class="das-cookie-banner__button-wrap das-cookie-banner__button-accept">
                             <button class="govuk-button das-cookie-banner__button" type="submit" data-accept-cookies="true">Accept all cookies</button>
                         </div>
                         <div class="das-cookie-banner__button-wrap das-cookie-banner__button-settings">
-                            <a class="govuk-button das-cookie-banner__button" role="button" id="btn-cookie-settings" 
-                            asp-external-controller="CookieConsent" asp-external-action="Settings" asp-external-subdomain="accounts">Set cookie preferences</a>
+                            <a class="govuk-button das-cookie-banner__button" role="button" asp-external-controller="CookieConsent" asp-external-action="Settings" asp-external-subdomain="accounts">Set cookie preferences</a>
```
Comment (shomavg) [inline src/SFA.DAS.Reservations.Web/Views/Shared/CookieBannerPartial.cshtml:13]: id has been removed here too.

Comment (Howesy77) [reply src/SFA.DAS.Reservations.Web/Views/Shared/CookieBannerPartial.cshtml]: Should be back now

**Pull request merged**